app/services/kredits.js

```diff
@@ -130,10 +130,15 @@ export default Service.extend({
   },
 
   addContributor(attributes) {
+    if (attributes.github_uid) {
+      const uidInt = parseInt(attributes.github_uid);
+      attributes.github_uid = uidInt;
+    }
+
     console.debug('[kredits] add contributor', attributes);
 
-    return this.kredits.Contributor.add(attributes)
-      .then((data) => {
+    return this.kredits.Contributor.add(attributes, { gasLimit: 350000 })
```

Comment:
as this is only for development we should set this manually in the metamask interface. 
I have the "Advanced gas controls" setting turned on - so the gas settings form is directly displayed when the transaction needs to be confirmed. This makes it easy to change them.

Reply:
Without this, I don't even get to the confirmation screen. The error happens beforehand, because the limit is too high.

Reply:
hmmm, will test it here again tomorrow. thought it worked for me, but maybe I was missing something. 

do you have an idea how we can best handle this? because so far it seems it is some development issue - and is not needed with rinkeby.

Reply:
It happens to me on Rinkeby.